example/weather/services/front/cmd/front/main.go

```diff
@@ -73,7 +73,7 @@ func main() {
 	// 3. Setup metrics
 	ctx = metrics.Context(ctx, genfront.ServiceName,
 		metrics.WithRouteResolver(func(r *http.Request) string {
-			return httptreemux.ContextRoute(r.Context())
+			return chi.RouteContext(r.Context()).RoutePattern()
```

Comment:
This actually does not work in the general case: the chi context gets initialized after this runs. Instead the code needs to be `mux.ResolvePattern(r)`: the Goa mux will initialize the mux context if not initialized already.